modules/bridge/src/main/ts/ephox/bridge/components/dialog/ImagePreview.ts

```diff
@@ -0,0 +1,26 @@
+import { FieldSchema, StructureSchema } from '@ephox/boulder';
+import { Optional } from '@ephox/katamari';
+
+import { FormComponent, formComponentFields, FormComponentSpec } from './FormComponent';
+
+export interface ImagePreviewSpec extends FormComponentSpec {
+  type: 'imagepreview';
+  width?: string;
+  height?: string;
+}
+
+export interface ImagePreview extends FormComponent {
+  type: 'imagepreview';
+  width: Optional<string>;
+  height: Optional<string>;
+}
+
+export const imagePanelSchema = StructureSchema.objOf(formComponentFields.concat([
+  FieldSchema.optionString('width'),
+  FieldSchema.optionString('height'),
+]));
+
+export const imagePanelDataProcessor = StructureSchema.objOf([
```

Comment:
These 2 still need renaming (not sure if you're finished renaming but just in case).

Reply:
I was, but there's even more than this I found. sigh :)

Reply:
OH it's because I had whole word search turned on. Right.